package.json

```diff
@@ -33,7 +33,7 @@
     "pre-push": "yarn format-check",
     "postinstall": "lerna bootstrap && yarn build && lerna link && npm run check-clean-workspace-after-install && opencollective-postinstall",
     "check-clean-workspace-after-install": "git diff --quiet --exit-code",
-    "test": "lerna run test --parallel",
+    "test": "lerna run test --concurrency 1",
```

Comment:
I was getting annoyed trying to decipher the test output.
because it ran them all in parallel the output was all mixed so if there was a failure, it was impossible to figure out the exact failure, and I'd have to go to that package and rerun the tests to find it out.
This will also make the CI output cleaner, as it suffered from the same problem.

It does mean the tests take slightly longer to run, but I think it's worth it.